user_docs/en/changes.md

```diff
@@ -5,7 +5,7 @@
 ### Important notes
 
 ### New Features
-
+* Support for math in PDFs has been added. This works for formulas that have MathML in an Associated File. Tagged formulas are supported in newer versions of TeX/LaTeX (9288, @NSoiffer)
```

Comment:
Can we add that this requires a supported PDF client?

Reply:
Will do.

Reply:
Also, write one sentence per line for the translation system.